applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
+Raw machine-translated text is a good mock-up to verify how the target language appears in localized content such as:
+* user interface,
+* images and diagrams.
+
+MT produces natural-language content that shall not be verified for accuracy at this step, but presents such features as target alphabet and typical word/sentence length. 
+When applied to localized content, MT allows to quickly issues, for example:
+* national characters corrupted or missing,
+* words or phrases truncated due to lack of screen space,
+* wrong sort order.
+Detecting these issues at early phase allows taking corrective actions, in parallel with the actual professional translation or post-editing, or planning for necessary post-processing of localized content, for example:
+* enable more character sets in localization (locale, font, etc.)
+* resize assets, or plan time and effort for resizing the assets with translated content,
+* enable sorting in the target locales.
+
+<!-- example of mock-up UI localization with national characters corrupted -->
+<!-- example of mock-up image localization with most of text truncated -->
+<!-- example of mock-up UI localization with wrong sorting of a translated list -->
+
+# Writing aid
+One can write fluently in their native language and use MT to draft text in the target language which they understand but need some help with correct expressions and phrases. 
```

Comment:
I would simplify this sentence to shorten it:

`Authors can use machine translation to draft their text translation.  When authors understand the target language, this technique can help to anticipate and fix potential translation problems.` 

What do you think?

Reply:
I edited this section to something in between your and my proposal. The idea is that there is no actual source text, it's only written down for a MT to propose nice sentences that one wouldn't think of :)

Reply:
Ah that makes sense!
I think I misinterpreted it then.